### ISSUE TITLE
Exec script can't use .Environment.Name arg

### ISSUE THREAD

**Issue by bradenwright** [roboll/helmfile#1220]
I'm trying to pass the environment as an arg to an exec.  So essentially:

```
mysetting: |
{{ exec "./mycmd" (list "default") | indent 2 }}
```
Works fine, but:
```
mysetting: |
{{ exec "./mycmd" (list .Environment.Name) | indent 2 }}
```
isn't working.  I'm sure there's something wrong with my syntax.  My sprig skills are more on the beginner side.

**Comment by mumoshu** [comment]
What exact result are you seeing? Is `.Environment.name` empty, or you get some within the script, or else?

**Comment by bradenwright** [comment]
I'll try to post a little more detail when I have time but its working now.  I think it may have been either (1) old version of helmfile, I've upgraded to latest or (2) I had code in a values.gotmpl file, I noticed that {{ .Environment.Name }} wasn't rendering in that file at all, so I moved code into helmfile.yaml.

B/t upgrading and moving code into helmfile.yaml its working.  Thanks for the quick reply
